templates/layout.html.twig

```diff
@@ -8,10 +8,15 @@
 
 {% endblock %}
 
+{% block javascripts %}
+    {{ parent() }}
+{% endblock %}
```

Comment:
there is no point in a block if you're only going to call the parent's content => if you want what's in the parent and nothing more, just don't add the block in your template

Reply:
resolved